aya/src/maps/bloom_filter.rs

```diff
@@ -120,10 +120,8 @@ mod tests {
 
     #[test]
     fn test_try_from_wrong_map() {
-        let map = new_map(test_utils::new_obj_map::<u32>(
-            BPF_MAP_TYPE_PERF_EVENT_ARRAY,
-        ));
-        let map = Map::PerfEventArray(map);
+        let map = new_map(test_utils::new_obj_map::<u32>(BPF_MAP_TYPE_ARRAY));
```

Comment:
Note to reviewers: This is necessary to stop miri tripping over the PERF_EVENT_ARRAY logic and attempting to `open` a file to read number of online CPUs

Reply:
That sounds like it should be a code comment.

Reply:
I agree. Let's make it a code comment before merging.